solaris/packer/solaris10.json

```diff
@@ -148,13 +141,13 @@
           "modifyvm",
           "{{.Name}}",
           "--memory",
-          "2048"
+          "8192"
```

Comment:
Maybe this is too much memory and we may lower it to 4GB, like the number of CPUs.

Reply:
As far as I know, Vagrant limit the amount of RAM assigned to each box, for example for Solaris in https://github.com/wazuh/wazuh-jenkins/blob/master/quality/deployments/vagrant/solaris/Vagrantfile#L141
This option is only used for the box build, I increased to be sure no memory issues are experimented during the build.  We could reduce it to 4 but it doesn't repercute on the final instances.